web/gds-admin-ui/src/components/ResendEmail.js

```diff
@@ -0,0 +1,91 @@
+
+import React from 'react'
+import { Modal, Button, Form } from 'react-bootstrap'
+import { useForm, useWatch } from "react-hook-form";
+import * as yup from 'yup';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+const schemaResolver = yupResolver(
+    yup.object().shape({
+        rejection_reason: yup.string().trim().test("test", 'Please enter the rejection email reason.', (value, context) => {
+            if (context.parent.email_type === "rejection" && !value) {
+                return false
+            }
+            return true
+        }),
+    })
+);
+
+
+
+function ResendEmail({ toggle, modal, vaspName }) {
+
+    const { register, handleSubmit, formState: { errors }, control } = useForm({
+        shouldUnregister: true, resolver: schemaResolver,
+        defaultValues: {
+            rejection_reason: '',
+            email_type: null
+        }
+    });
+    const [, setResult] = React.useState("");
+    const emailType = useWatch({
+        control, name: "email_type"
+    })
+
+    const onSubmit = (data) => {
+        setResult(JSON.stringify(data))
+    };
+
+
+    return (
+        <div>
+            <Modal show={modal} dialogClassName="modal-right">
+                <Form noValidate onSubmit={handleSubmit(onSubmit)}>
+                    <Modal.Header onHide={toggle} closeButton>
+                        <h4 className="modal-title">Resend email</h4>
+                    </Modal.Header>
+                    <Modal.Body>
+                        <h5 className="mb-3">{vaspName}</h5>
+                        <p>What kind of email do you want to send ?</p>
+                        <div className="">
+                            <div>
+                                <Form.Label htmlFor="verifyContact" />
+                                <Form.Check {...register("email_type")} id="verifyContact" type="radio" value="verify contact" label="Resend verify contact email" required />
+
+                                <Form.Label htmlFor="review" />
+                                <Form.Check {...register("email_type")} id="review" type="radio" value="review request" label="Resend review request email" required />
+
+                                <Form.Label htmlFor="certificateDelivrey" />
+                                <Form.Check {...register("email_type")} id="certificateDelivrey" type="radio" value="certificate delivery" label="Resend certificate delivery email" required />
```

Comment:
Please update to `label="Redeliver certificates" value="deliver_certs"`

Reply:
If it's possible to add help text to the radio button, it should be "Sends PKCS12 encrypted certs to technical contact if still available".